tests/roles/pull_openstack_configuration/tasks/main.yaml

```diff
@@ -1,3 +1,52 @@
+# OS Diff automation steps
+- name: Make sure Golang is installed
+  failed_when: false
+  become: true
+  ansible.builtin.shell: |
+    dnf -y install golang
```

Comment:
We should only execute this if `go` binary is not found? IIRC most dev setups install Go differently than from the RPM.